### DISCUSSION TITLE
My Affiliation status can't be verified, although I already sent every possible document.

### DISCUSSION THREAD

**Discussion by NotJansel** [community/community#56164] [GitHub Education]
### Hi everyone,

Good evening,

I now tried 3 times to verify for the student developer pack, twice with my school id that apparently didnt work and once with my last report card.

The email always states: 
- Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

or 

- You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

- Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

- Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

while the images provided (except one image I sent from my school's iserv portal) were all done with my phones camera.

any help would be appreciated, thank you!

**Comment by samus-aran** [comment]
Hi @NotJansel 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

**Reply by NotJansel** [reply]
Well, I basically tried Everything I have from my school.
